codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/protocols/RestJson1.kt

```diff
@@ -618,6 +618,8 @@ class RestJson1HttpDeserializerGenerator(
                 "let request = request.map(|body| #{Bytes}::from(body.bytes().unwrap().to_vec()));",
                 *codegenScope
             )
+            rust("assert_eq!(request.method().as_str(), \"${testCase.method}\");")
+            rust("assert_eq!(request.uri().to_owned(), \"${testCase.uri}\");")
```

Comment:
(you'll need to import it)
```suggestion
            rust("assert_eq!(request.method().as_str(), ${testCase.method.dq()});")
            rust("assert_eq!(request.uri().to_owned(), ${testCase.uri.dq()});")
```